universite-du-quebec-a-montreal-prenoms.csl

```diff
@@ -67,91 +67,87 @@
     <choose>
       <if type="article-journal book chapter paper-conference entry-encyclopedia entry-dictionary" match="none">
         <names variable="editor" delimiter=", " prefix=" (" suffix=")">
-          <name and="symbol" initialize="false"/>
+          <name delimiter-precedes-last="never" initialize="false" name-as-sort-order="all"/>
           <label form="short" prefix=", "/>
         </names>
       </if>
     </choose>
   </macro>
   <macro name="author">
-    <choose>
-      <if match="any" variable="composer">
-        <names variable="composer">
-          <name and="text" delimiter-precedes-last="never" et-al-use-last="true" initialize="false" name-as-sort-order="all"/>
-        </names>
-      </if>
-      <else>
+    <names variable="composer" delimiter=", ">
+      <name delimiter-precedes-last="never" name-as-sort-order="all"/>
+      <substitute>
         <names variable="author">
-          <name and="text" delimiter-precedes-last="never" et-al-use-last="true" initialize="false" name-as-sort-order="all"/>
-          <label form="short" prefix=" (" suffix=")"/>
-          <substitute>
-            <choose>
-              <if type="entry-dictionary entry-encyclopedia webpage manuscript legislation legal_case bill treaty article map article-magazine article-newspaper" match="any">
-                <text macro="title"/>
-              </if>
-              <else-if type="book" match="any">
-                <names variable="editor"/>
-                <text macro="title" font-style="italic"/>
-              </else-if>
-              <else>
-                <names variable="editor"/>
-                <names variable="translator"/>
-              </else>
-            </choose>
-            <choose>
-              <if type="report">
-                <text variable="publisher"/>
-                <text variable="title" font-style="italic" suffix="."/>
-              </if>
-            </choose>
-          </substitute>
+          <name delimiter-precedes-last="never" name-as-sort-order="all"/>
         </names>
-      </else>
+        <names variable="illustrator"/>
+        <names variable="director" delimiter=", ">
+          <name delimiter-precedes-last="never" name-as-sort-order="all"/>
```

Comment:
you can just use 
`<names variable="director"/>` 
The delimiter isn't doing anything if there's on a single variable and the `name` attributes are inherited in substitute.